packages/protocol/test/governance/voting/lockedgold.ts

```diff
@@ -203,11 +203,10 @@ contract('LockedGold', (accounts: string[]) => {
         })
 
         it('should add a pending withdrawal', async () => {
-          const [values, timestamps] = await lockedGold.getPendingWithdrawals(account)
-          assert.equal(values.length, 1)
-          assert.equal(timestamps.length, 1)
-          assertEqualBN(values[0], value)
-          assertEqualBN(timestamps[0], availabilityTime)
+          // Get the first pending withdrawal.
+          const [val, timestamp] = await lockedGold.getPendingWithdrawal(account, 0)
+          assertEqualBN(val, value)
+          assertEqualBN(timestamp, availabilityTime)
```

Comment:
Maybe add an assertReverts here for getting withdrawal with index 1 to
1. test that your function fails when given an out of bounds index
2. test that `unlock` indeed adds exactly one withdrawal (as was tested previously when checking the list's length)